packages/react/test/specs/components/Slider/Slider-test.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react'
+import { ReactWrapper } from 'enzyme'
+
+import { mountWithProvider as mount } from 'test/utils'
+import { isConformant, handlesAccessibility } from 'test/specs/commonTests'
+import Slider from 'src/components/Slider/Slider'
+
+const testValue = '30'
+const getInputDomNode = (sliderComp: ReactWrapper): HTMLInputElement =>
+  sliderComp.find('input').getDOMNode() as HTMLInputElement
+
+const setUserInputValue = (sliderComp: ReactWrapper, value: string) => {
+  sliderComp.find('input').simulate('change', { target: { value } })
+}
+
+describe('Slider', () => {
+  describe('conformance', () => {
```

Comment:
```suggestion
```

We don't need a separate `describe()` there

Reply:
done, thanks